src/components/Header/HeaderActions.tsx

```diff
@@ -33,7 +39,23 @@ function HeaderActions (props) {
         <div className="flexStretch"/>
 
         <div className="navPadding"> 
-            <HeaderWalletConnector/>
+            {account ? (
+                <Button 
+                    // href="/"
+                    onClick={toggleWalletDropdown}
```

Comment:
what do you think about making this disconnect the wallet? that was a request from one of the other guys in the group, that you be able to disconnect in the UI vs requiring you to use metamask to disconnect